backend/app/usecase/validator/alias.go

```diff
@@ -13,20 +16,21 @@ type CustomAlias struct {
 
 // IsValid checks whether the given alias has valid format.
 func (c CustomAlias) IsValid(alias *string) bool {
-	if alias == nil {
-		return true
-	}
-
-	if *alias == "" {
+	if alias == nil || *alias == "" {
```

Comment:
Prefer separate if & else for simplicity

Reply:
Done